com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -756,6 +759,7 @@ internal static void Sanitize(ref FrameSettings sanitizedFrameSettings, Camera c
             // TODO: The work will be implemented piecemeal to support all passes
             bool msaa = sanitizedFrameSettings.bitDatas[(int)FrameSettingsField.MSAA] &= renderPipelineSettings.supportMSAA && sanitizedFrameSettings.litShaderMode == LitShaderMode.Forward && !pipelineSupportsRayTracing;
             sanitizedFrameSettings.bitDatas[(int)FrameSettingsField.AlphaToMask] &= msaa;
+            sanitizedFrameSettings.bitDatas[(int)FrameSettingsField.LitEmissiveAsForward] &= sanitizedFrameSettings.litShaderMode == LitShaderMode.Deferred;
```

Comment:
I see above that the emissive pass feature doesn't work if the frame settings `OpaqueObjects` is disabled. Maybe it's worth to add it here in the sanitize?